x-pack/legacy/plugins/siem/public/pages/detection_engine/index.tsx

```diff
@@ -7,6 +7,8 @@
 import React, { useEffect } from 'react';
 import { Redirect, Route, Switch, RouteComponentProps } from 'react-router-dom';
 
+import { useKibana } from '../../../../../../../src/plugins/kibana_react/public';
```

Comment:
Can you import this from `../../lib/kibana` instead? That one's typed to include our dependencies, and gives us room to change the implementation.